CAP/gap/InstallAdds.gi

```diff
@@ -233,6 +221,12 @@ InstallGlobalFunction( CapInternalInstallAdd,
             fi;
             
         od;
+
+        display_error_for_current_function := function( message )
```

Comment:
You could also move this function outside of `CapInternalInstallAdd`, and give the other parameters as arguments. This way you could prevent it from being created a lot of times, but only once.

Reply:
Okay, should be fixed

Reply:
Thanks :)